### PR TITLE
Fix dapp connection

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

File: src/composables/auth.ts [hunk 1]
```diff
@@ -218,6 +218,8 @@ export const useAuth = createCustomScopedComposable(() => {
       const interval = setInterval(async () => {
         const sessionEncryptionKey = await getSessionEncryptionKey();
         if (sessionEncryptionKey) {
+          const decryptionResult = await decrypt(sessionEncryptionKey, mnemonicEncrypted.value!);
+          mnemonicDecrypted.value = decryptionResult;
```
Comment (peronczyk) [inline src/composables/auth.ts:222]:
```suggestion
          mnemonicDecrypted.value = await decrypt(sessionEncryptionKey, mnemonicEncrypted.value!);
```

File: src/popup/App.vue [hunk 2]
```diff
@@ -14,7 +14,7 @@
         class="app-inner"
         :class="{ 'styled-scrollbar': showScrollbar }"
       >
-        <Header v-if="isAuthenticated && showHeader" />
+        <Header v-if="showHeader" />
```
Comment (peronczyk) [inline src/popup/App.vue:17]: There was a reason to hide the header when user is not authenticated - previously user was seeing for a while the remnants of the header on the login screen.

**Review by peronczyk — APPROVED** [review]

File: src/popup/App.vue (hunk 2 above)
Comment (CedrikNikita) [reply src/popup/App.vue:17]: This remnants were fixed in a different way by waiting for router to be ready.
`isRouterReady` variable is added into equation of `showHeader` computed variable.

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Pull request merged**